.projen/tasks.json

```diff
@@ -366,7 +366,7 @@
       "description": "Run tests",
       "steps": [
         {
-          "exec": "jest --passWithNoTests --coverageProvider=v8 --updateSnapshot",
+          "exec": "jest --passWithNoTests --updateSnapshot",
```

Comment:
Update the `jest` command to include the `--coverageProvider=v8` flag to ensure proper code coverage reporting. This is already reflected in the `package.json` file. [medium]